backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -155,6 +165,15 @@ public EndpointPage getEndpoints(
             count = endpointRepository.getEndpointsCount(principal.getAccount(), name);
         }
 
+        for (Endpoint endpoint: endpoints) {
+            if (isForOpenBridge(endpoint)) {
+                // Don't return the processor info, it is internal only
```

Comment:
Do we have a test that verifies these fields are never returned to the client?

Reply:
No. As they are removed here, they should not be returned. I am probably missing something?

Reply:
The current code is fine, you didn't miss anything. What I meant is that we could add a test to make sure this information isn't leaked unintentionally in the future.